pkg/pillar/cmd/zedmanager/zedmanager.go

```diff
@@ -216,6 +219,33 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject, ar
 	ctx.subAppInstanceConfig = subAppInstanceConfig
 	subAppInstanceConfig.Activate()
 
+	// Get AppInstanceConfig from zedmanager itself
+	subLocalAppInstanceConfig, err := ps.NewSubscription(pubsub.SubscriptionOptions{
```

Comment:
Somewhere we should document what these are about (basically tracking the AppInstanceConfig which we have just rolled back to until the controller/zedagent sends us that as AppInstanceConfig) and that they take precedence over the AppInstanceConfig received from the controller/zedagent. 

I don't know if this is the best place to do that.

Note that we also have pkg/pillar/docs/zedmanager.md (I hope), and we can use that to add high-level descriptions about snapshot and rollback.

Reply:
We will rework the rolling back mechanism soon so that the controller sends us the target (snapshotted) config immediately with the rollback command. It may change the logic of this subscription. So, I would prefer to document it after we finish the rollback command's final version.

Reply:
OK will you do that rework as part of this PR or as a follow-on PR?

Reply:
It will be part of a new PR.
We need this version to be merged to have at least the initial version of the snapshot feature working in the master branch. After that, we will start with the rework on Cloud-side and then - EVE.